index.js

```diff
@@ -235,7 +246,8 @@ Pjax.prototype = {
       window.history.pushState({
           url: state.href,
           title: state.options.title,
-          uid: this.maxUid
+          uid: this.maxUid,
+          scrollPos: 0
```

Comment:
This also should be `[0, 0]`, per my other comment

Reply:
Good point.